src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -327,6 +328,7 @@ export function ObjectSummary({
     };
 
     const onOpenPreview = () => {
+        dispatch(api.util.invalidateTags(['PreviewData']));
```

Comment:
I think we shouldn't cache this request at all. Every time we click on preview, data should be requested.

Reply:
Any ideas of other ways how it can be achieved despite tag invalidation?